examples/MassStorage/msc_ramdisk_dual/ramdisk.h

```diff
@@ -25,30 +25,25 @@
 #ifndef RAMDISK_H_
 #define RAMDISK_H_
 
-//--------------------------------------------------------------------+
-// LUN 0
-//--------------------------------------------------------------------+
-#define README0_CONTENTS \
-"LUN0: This is tinyusb's MassStorage Class demo.\r\n\r\n\
-If you find any bugs or get any questions, feel free to file an\r\n\
-issue at github.com/hathach/tinyusb"
+#define README_CONTENTS \
+"This is Adafruit TinyUSB MassStorage device demo on RAM disk."
 
-uint8_t msc_disk0[DISK_BLOCK_NUM][DISK_BLOCK_SIZE] =
+uint8_t msc_disk[DISK_BLOCK_NUM][DISK_BLOCK_SIZE] =
```

Comment:
please revert to the previous version of ramdisk.h in this dual lun example. It is intended to show up as `TinyUSB 0` and `TinyUSB 1` with different readme file contents as well